client-go/controller/client/client.go

```diff
@@ -26,13 +26,20 @@ type Client struct {
 
 	// Username is the name of the user performing requests against the API.
 	Username string
+
+	// ResponseLimit is the number of results to return on a paginated requests.
```

Comment:
"paginated" might confuse here.  The controller supports it, but the CLI can only ever fetch one "page," even though its size is specifiable.  Same comment applies on line 34.